### ISSUE TITLE
Clarification (issues) with starting using V2

### ISSUE THREAD

**Issue by janblahovec** [aquasecurity/postee#169]
Hello there!

Today I started to configure and use postee v2.0.3 and I have some questions for you:
1. I saw that configuration of rego files are a bit different as was in version 1. For example: In my use case I had several rego files, which were checking response from Aquasec image scan. Based on image name (repository) I was sending response to concrete MS teams channel. 
Example:

cfg.yaml
```
- name: msteam-teamA
  type: teams
  enable: true
  url: <url to concrete channel>
  Policy-Non-Compliant: true 
  Policy-OPA: ["../rego/teamA-policy.rego"]
```
../rego/teamA-policy.rego
```
package postee
default allow = false
allow {
    regex.match("teamA/*", input.image)
}
allow {
    regex.match("teamAa-*", input.image)
}
```
Right now configuration is in a bit different way. 
cfg.yaml
```
- name: msteam-naf
  input: |
    contains(input.image, "teamA/")
    contains(input.image, "teamAa-")
  outputs: [ msteam-teamA ]
  template: legacy

...
- name: msteam-teamA
  type: teams
  enable: true
  url: <url to concrete MS teams channel>
```
So, could you confirm if this "conversion" to v2 is good? Or how could I check if concrete scanned image name is teamA/... OR teamAa- ?

2. There was default format of message in V1 , which could be sent to MS Teams and looks like this output:
![image](https://user-images.githubusercontent.com/45173032/132708774-3999d90d-5fac-4a0f-884e-993cbca928c1.png)
There is a red square in bottom of the screenshot. When I clicked on that link, I was redirected to Aquasec scan result of this concrete scan. But in V2 this link points me to https://<url to Aquasec>/#/images/ , so to the list of all scans. Is this a bug, or did I misconfigure something?

Many thanks for your answers!

I wish you a nice day.
Best regards,
Jan Blahovec

**Comment by jerbia** [comment]
@janblahovec 

1. The conversion looks OK. As for the "OR" statement - it's a bit challenging in REGO. You can either create two separate route entries, each one with one matching rule, or you can create a more complex "regex.match" rule that has a logical "OR" as part of the regexp. I think it will be simpler (and more readable) to create two routing rules.

2. @AndreyLevchenko is this a bug (where the Teams scan results are not filling the Aqua URL)?

**Comment by AndreyLevchenko** [comment]
@janblahovec @jerbia 
I suspect it may be linked with property name change. In v2 all properties follow similar name convention so `AquaServer`  becomes `aqua-server`

**Comment by jerbia** [comment]
That's right - the property names have changed to <all-lower>-<all-lower>.

@janblahovec can you please check and let us know if it works?

**Comment by janblahovec** [comment]
Hello @jerbia @AndreyLevchenko !

regarding the 1. point - ok I will do probably more routes, but I think ( my personal opinion) this was done nicely in V1. We have teams which are responsible for many projects. So, when it was configured in previous version, I had one rego file per team and there were many rules in each rego file. Then I configured only path to that rego file in cfg.yaml. 

2. I have `aqua-server` property in my cfg.yaml file

Thanks.
BR,
Jan Blahovec

**Comment by janblahovec** [comment]
Hello @jerbia @AndreyLevchenko 

regarding the wrong url
![image](https://user-images.githubusercontent.com/45173032/133224376-52b8a2bd-0e19-457e-8265-039099b5cb03.png)

I have url https://<URL_TO_MY_AQUASEC>/**#/images/** n Teams notification.
Url is good, but it is pointing to all images, not to concrete scan. E.g. I had url in V1 like this https://<URL_TO_MY_AQUASEC>/**#/images/Converge%20DEV%20ECR/b360%2Fb360-uaa:fd28c097d456d0cbe26f91dda71e2f8d3313c1ed**

BR,
Jan

**Comment by AndreyLevchenko** [comment]
@janblahovec 
I added my fixes to `main` branch. Can you check if issue is resolved?

**Comment by janblahovec** [comment]
@AndreyLevchenko 
So, shall I build an image from `main` branch?

**Comment by AndreyLevchenko** [comment]
@janblahovec 
We would appreciate if you could test the changes before release.
Please use `Dockerfile` to build Postee image. 

Also after discussion with @jerbia we decided to add support for opa files. I will create separate issue for that

**Comment by janblahovec** [comment]
@AndreyLevchenko 
Hello. I tried this.. and it is working as it should be.
Please release an official version and then let me know.
Thanks.
BR,
Jan

**Comment by janblahovec** [comment]
@AndreyLevchenko @jerbia Any updates here?
